infrastructure/terraform/modules/gcp-gke/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "project_id" {
+  type        = string
+  default     = "airy-core"
+  description = "The project defined in gcloud config is airy-core"
+}
+variable "region" {
+  type        = string
+  default     = "us-central1"
+  description = "The region defined in gcloud config is us-central1"
+}
+variable "zone" {
+  type        = string
+  default     = "us-central1-a"
+  description = "The zone defined in gcloud config is us-central1-a"
+}
+variable "gke_num_nodes" {
+  default     = 2
+  description = "number of gke nodes"
+}
+variable "vpc_name" {
+  default = "airy-core-vpc"
+}
+variable "private_subnets" {
```

Comment:
The subnets are not used, so we can also remove the variables.

Reply:
👍🏼